pyfar/classes/audio.py

```diff
@@ -706,8 +706,8 @@ def __init__(
 
         # check fft norm
         if fft_norm in self._VALID_FFT_NORMS:
-            if self._complex and fft_norm in ["rms", "power"]:
-                raise ValueError((f"'{fft_norm}' normalization is not valid "
+            if self._complex and fft_norm in ["rms", "power", "psd"]:
+                raise ValueError((f"'{fft_norm} normalization is not valid "
```

Comment:
Lets maybe give a full list of non-valid norms. Otherwise one might change the norm to get the same error again:
`'The rms, power, and psd FFT normalizations are not valid for complex signals'`